.travis.yml

```diff
@@ -1,13 +1,29 @@
+# by default Travis uses JDK 8u31, which is much too old for Scala 2.12 (versions before 8u102 have issues - https://issues.scala-lang.org/browse/SI-9828)
+# Trusty has 8u131 or later
+dist: trusty
+sudo: required
+
 language: scala
-script: sbt -jvm-opts jvmopts.travis -Dslick.testkit-config=test-dbs/testkit.travis.conf +testAll
+
+scala:
+  - 2.12.3
+  - 2.11.11
+  - 2.10.6
+
+script: sbt ++$TRAVIS_SCALA_VERSION -jvm-opts jvmopts.travis -Dslick.testkit-config=test-dbs/testkit.travis.conf +testAll
+
 jdk:
-  - openjdk6
+  - oraclejdk8
```

Comment:
We can't just switch to Java 8 for all builds. The Scala 2.10 and 2.11 releases have to be able to run on Java 6, so we need to test that. I would use a build matrix with openjdk6 and oraclejdk8 in Travis, and then in the sbt build check for the Java version and set the cross Scala versions accordingly (2.10.6 and 2.11.11 on Java 6, 2.12.3 on Java 8).

Reply:
Got nice error message:
> Sun/Oracle JDK 6 is EOL since November 2012, and is no longer supported. Please consider upgrading...
Don't know if I should laugh or cry :)

Reply:
using `openjdk6` instead